mgplvm/lpriors/euclidean.py

```diff
@@ -61,9 +61,9 @@ def __init__(self,
         self.n_samples = n_samples
         self.d = d
         #1d latent and n_z inducing points
-        zinit = torch.linspace(0., torch.max(ts), n_z).reshape(1, 1, n_z)
+        zinit = torch.linspace(0., torch.max(ts).item(), n_z).reshape(1, 1, n_z)
         #separate inducing points for each latent dimension
-        z = InducingPoints(n, 1, n_z, z=zinit.repeat(n, 1, 1))
+        z = InducingPoints(n, d, n_z, z=zinit.repeat(n, d, 1))
```

Comment:
Is there a reason you want separate inducing points per trial? It forces us to invert the kernel separately for each  trial instead of once for all of them does it not?

Reply:
Here `n` is the dimension of the latents and `d` is the dimension of "time", which should be 1 in most cases. We don't have separate inducing points per trial?

Reply:
ah right, my bad!